target/linux/ramips/dts/mt7628an_wavlink_wl-wn576a2.dts

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "wavlink,wl-wn576a2", "silvercrest,SWV-733-B1", "mediatek,mt7628an-soc";
+	model = "WAVLINK WL-WN576A2";
+
+	aliases {
+		led-boot = &led_wps;
+		led-failsafe = &led_wps;
+		led-running = &led_wps;
+		led-upgrade = &led_wps;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 43 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		repeatermode {
+			label = "repeatermode";
+			gpios = <&gpio 41 GPIO_ACTIVE_HIGH>;
+			linux,code = <BTN_0>;
+		};
+
+		accesspointmode {
+			label = "accesspointmode";
+			gpios = <&gpio 42 GPIO_ACTIVE_HIGH>;
+			linux,code = <BTN_1>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+               wifi-excellent { //453/448
+                        label = "blue:wifi-excellent";
+                        gpios = <&gpio 37 GPIO_ACTIVE_LOW>;
+                };
+
+               wifi-high { //491/480
+                        label = "blue:wifi-high";
+                        gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+                };
+
+               wifi-med { //460/448
+                        label = "blue:wifi-med";
+                        gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
+                };
+
+                wifi-low {
+                        label = "blue:wifi-low";
+                        gpios = <&gpio 39 GPIO_ACTIVE_LOW>;
+                };
+
+                wifi-poor {
+                        label = "blue:wifi-poor";
```

Comment:
Is this "excellent" and "poor" level introduced by the vendor?

Reply:
No, if any another wifi strength scale exists I will use it